package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -8,6 +8,7 @@ PROG=/usr/sbin/dnsmasq
 
 ADD_LOCAL_DOMAIN=1
 ADD_LOCAL_HOSTNAME=1
+ADD_LOCAL_GATENAME=0
```

Comment:
Yes, better explanation. I took `LOCAL_HOSTNAME` to mean your internal (business) network. If your exterior WAN however had a good name (or a few) in global DNS, then you may not want that sides name changed by this option. That would be for business class service from ISP. `LOCAL_GATENAME` allows you to split it. For most people with residential service on a consumer router, you will just set both true.